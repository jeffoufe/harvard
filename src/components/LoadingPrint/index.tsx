import React from 'react';
import { Grid, Paper } from '@material-ui/core';
import useStyles from '../Print/styles';
import { Skeleton } from '@material-ui/lab';

const LoadingPrint = () => {
    const classes = useStyles();
    return (
        <Grid item xs={12} sm={6} md={4} lg={3}>
            <Paper className={classes.paper}>
                <Skeleton variant='rect' height={200} />
                <Skeleton variant='text' className={classes.legend} />
                <Skeleton variant='text' />
                <Skeleton variant='text' />
            </Paper>
        </Grid>
    );
}

LoadingPrint.displayName = 'LoadingPrint';
export default LoadingPrint;