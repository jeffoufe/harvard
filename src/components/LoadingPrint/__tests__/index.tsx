import React from 'react';
import { shallow } from 'enzyme';
import LoadingPrint from '..'; 

describe('LoadingPrint', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<LoadingPrint />);
        expect(wrapper).toMatchSnapshot();
    })
})