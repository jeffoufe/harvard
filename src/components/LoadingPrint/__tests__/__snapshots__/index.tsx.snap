// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPrint renders correctly 1`] = `
<WithStyles(ForwardRef(Grid))
  item={true}
  lg={3}
  md={4}
  sm={6}
  xs={12}
>
  <WithStyles(ForwardRef(Paper))
    className="makeStyles-paper-1"
  >
    <WithStyles(ForwardRef(Skeleton))
      height={200}
      variant="rect"
    />
    <WithStyles(ForwardRef(Skeleton))
      className="makeStyles-legend-4"
      variant="text"
    />
    <WithStyles(ForwardRef(Skeleton))
      variant="text"
    />
    <WithStyles(ForwardRef(Skeleton))
      variant="text"
    />
  </WithStyles(ForwardRef(Paper))>
</WithStyles(ForwardRef(Grid))>
`;
